packages/block-library/src/image/edit.js

```diff
@@ -461,12 +472,28 @@ class ImageEdit extends Component {
 								imageHeight,
 							} = sizes;
 
+							const defaultAlt = sprintf( __( 'This image has an empty alt attribute; its file name is %s' ), this.getFilename( url ) );
 							// Disable reason: Image itself is not meant to be
 							// interactive, but should direct focus to block
 							// eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
 							const img = <img src={ url } alt={ alt } onClick={ this.onImageClick } />;
+							// Disable reason: Image itself is not meant to be
+							// interactive, but should direct focus to block
+							// eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+							const imgNoAlt = <img src={ url } alt={ defaultAlt } onClick={ this.onImageClick } />;
```

Comment:
I think this could have been done in a much smaller change, something along the lines of:
```
const defaultedAlt = alt ? alt : sprintf( __( 'This image has an empty alt attribute; its file name is %s' ), this.getFilename( url ) );
const img = <img src={ url } alt={ defaultedAlt } onClick={ this.onImageClick } />;
```
Would avoid the need for the if statement on line 486 and the ternary on 574.

Reply:
great suggestion. that makes this much more readable.